aws-ssm-document/src/main/java/com/amazonaws/ssm/document/UpdateHandler.java

```diff
@@ -152,4 +167,8 @@ private OperationStatus getOperationStatus(@NonNull final ResourceStatus status)
                 return OperationStatus.FAILED;
         }
     }
+
+    private int setCallbackDelay(final OperationStatus operationStatus) {
```

Comment:
can we have a helper method to be reused by both Create/Update handlers?

Reply:
yeah I kind of wanted to keep them separate because if we have a common class with methods for different operations, those methods would be filled with If conditions as the requirements start diverging depending on the operations create/update/delete/list. So I just kept them separate and let the individual handlers handle for now. We can certainly move them together as the code grows and if it'd make more sense then.